src/components/schedule-page/schedule-editing.component.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Button } from "../common-components";
+import ValidationDrawerComponent from "./validation-drawer/validation-drawer.component";
+import backend from "../../api/backend";
+import { ScheduleErrorActionType } from "../../state/reducers/schedule-errors.reducer";
+import { ActionModel } from "../../state/models/action.model";
+import { ScheduleError } from "../../common-models/schedule-error.model";
+import { useDispatch, useSelector } from "react-redux";
+import { ApplicationStateModel } from "../../state/models/application-state.model";
+import { ScheduleComponent } from "./table/schedule/schedule.component";
+import UndoIcon from "@material-ui/icons/Undo";
+import RedoIcon from "@material-ui/icons/Redo";
+
+export function ScheduleEditingComponent(): JSX.Element {
+  const schedule = useSelector((state: ApplicationStateModel) => state.scheduleData?.present);
+  const dispatcher = useDispatch();
+
+  async function updateScheduleErrors(): Promise<void> {
+    if (schedule) {
+      const response = await backend.getErrors(schedule);
+      dispatcher({
+        type: ScheduleErrorActionType.UPDATE,
+        payload: response,
+      } as ActionModel<ScheduleError[]>);
+    }
+  }
+
+  function Buttons(): JSX.Element {
+    return (
+      <div id={"edit-page"}>
+        <div className={"editing-row"}>
+          <div className={"buttons"}>
+            <div className={"filler"} />
+
+            <div id={"edit-panel-text-container"}>
+              <p>Tryb edycji aktywny</p>
+            </div>
+            <Button variant="circle-outlined">
+              <RedoIcon className={"edit-icons"} />
```

Comment:
Kolejność przycisków na odwrót. Zawaliliśmy to w opisie zadania. Mógłbyś to jeszcze poprawić w tym PR?